package.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "strong-config-ts",
```

this should be the npm package name to avoid issues

```suggestion
  "name": "@strong-config/node",
```